typescript/github-proxy/.dev.vars.example

```diff
@@ -0,0 +1 @@
+GITHUB_API_KEY=
```

Comment:
I'm not familiar with the `.dev.vars` file. It is like `.env` files?

Reply:
yes, but its specifically for Workers